go/multiorch/init.go

```diff
@@ -51,7 +60,11 @@ func (mo *MultiOrch) RunDefault() {
 // Register flags that are specific to multiorch.
 func (mo *MultiOrch) RegisterFlags(fs *pflag.FlagSet) {
 	fs.String("cell", mo.cell.Default(), "cell to use")
-	viperutil.BindFlags(fs, mo.cell)
+	fs.StringSlice("shard_watch_targets", mo.shardWatchTargets.Default(), "list of db/tablegroup/shard targets to watch")
+	fs.Duration("bookkeeping_interval", mo.bookkeepingInterval.Default(), "interval for bookkeeping tasks")
+	fs.Duration("cluster_metadata_refresh_interval", mo.clusterMetadataRefreshInterval.Default(), "interval for refreshing cluster metadata from topology")
+	fs.Duration("cluster_metadata_refresh_timeout", mo.clusterMetadataRefreshTimeout.Default(), "timeout for cluster metadata refresh operation")
+	viperutil.BindFlags(fs, mo.cell, mo.shardWatchTargets, mo.bookkeepingInterval, mo.clusterMetadataRefreshInterval, mo.clusterMetadataRefreshTimeout)
```

Comment:
We might need a custom linter to prevent underscores in flags. All flags need to use only dashes.